patches/node/ci_ensure_node_tests_set_electron_run_as_node.patch

```diff
@@ -0,0 +1,72 @@
+index b09eeeb9971cf6dfbdccc700757487fe2236e5a8..dd8296e63133eec6babac9b35e77da301a44b86f 100644
+--- a/test/fixtures/test-runner/output/arbitrary-output-colored.js
++++ b/test/fixtures/test-runner/output/arbitrary-output-colored.js
+@@ -6,6 +6,6 @@ const fixtures = require('../../../common/fixtures');
+ 
+ (async function run() {
+   const test = fixtures.path('test-runner/output/arbitrary-output-colored-1.js');
+-  await once(spawn(process.execPath, ['--test', test], { stdio: 'inherit', env: { FORCE_COLOR: 1 } }), 'exit');
+-  await once(spawn(process.execPath, ['--test', '--test-reporter', 'tap', test], { stdio: 'inherit', env: { FORCE_COLOR: 1 }  }), 'exit');
+-})().then(common.mustCall());
++  await once(spawn(process.execPath, ['--test', test], { stdio: 'inherit', env: { ELECTRON_RUN_AS_NODE: 1, FORCE_COLOR: 1 } }), 'exit');
++  await once(spawn(process.execPath, ['--test', '--test-reporter', 'tap', test], { stdio: 'inherit', env: { ELECTRON_RUN_AS_NODE: 1, FORCE_COLOR: 1 }  }), 'exit');
++})().then(common.mustCall());
+\ No newline at end of file
+diff --git a/test/parallel/test-node-output-console.mjs b/test/parallel/test-node-output-console.mjs
+index 5a1b9feb6c8bedb50b89f5c4f3c5983455bb042d..efca7811dc0b6a590c5ee023c71801703a642882 100644
+--- a/test/parallel/test-node-output-console.mjs
++++ b/test/parallel/test-node-output-console.mjs
+@@ -31,6 +31,7 @@ describe('console output', { concurrency: true }, () => {
+     .transform(snapshot.replaceWindowsLineEndings, snapshot.replaceWindowsPaths, replaceStackTrace);
+   for (const { name, transform, env } of tests) {
+     it(name, async () => {
++      if (env) env.ELECTRON_RUN_AS_NODE = 1;
+       await snapshot.spawnAndAssert(fixtures.path(name), transform ?? defaultTransform, { env });
+     });
+   }
+diff --git a/test/parallel/test-node-output-errors.mjs b/test/parallel/test-node-output-errors.mjs
+index fca2149fea32123dc4d4c6f21c64cc3c583454b3..b9a55fb7ea22e62553f69bd035797f7aaee1fc38 100644
+--- a/test/parallel/test-node-output-errors.mjs
++++ b/test/parallel/test-node-output-errors.mjs
+@@ -53,6 +53,7 @@ describe('errors output', { concurrency: true }, () => {
+     !skipForceColors ? { name: 'errors/force_colors.js', env: { FORCE_COLOR: 1 } } : null,
+   ].filter(Boolean);
+   for (const { name, transform, env } of tests) {
++    if (env) env.ELECTRON_RUN_AS_NODE = 1;
+     it(name, async () => {
+       await snapshot.spawnAndAssert(fixtures.path(name), transform ?? defaultTransform, { env });
+     });
+diff --git a/test/pseudo-tty/test-assert-colors.js b/test/pseudo-tty/test-assert-colors.js
+index 9d5a923aa0f8e7104b17fdd214d4ebf4c715cf11..2459000d8e367e181a978533fa85f0db18f7de57 100644
+--- a/test/pseudo-tty/test-assert-colors.js
++++ b/test/pseudo-tty/test-assert-colors.js
+@@ -3,6 +3,7 @@ require('../common');
+ const assert = require('assert').strict;
+ 
+ assert.throws(() => {
++  process.env.ELECTRON_RUN_AS_NODE = '1';
```

Comment:
We probably don't need this since it's not germane to the actual test

Reply:
confirmed; test-assert-colors passes without that variable. Removed in 0ea753babd37ca1710a468c6d2dd9b115ff1feb1.